patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -615,42 +622,46 @@ function commonCloseMenuTasks() {
 }
 
 function openMenu() {
+  let lockPage = function () {
+    document.querySelector("body").style.top = `-${alertlOffsetPosition}px`;
+    document.querySelector("body").style.position = "fixed";
```

Comment:
@ygannett On Desktop browsers, this causes the body width to change. It seems like setting the width to 100vw fixed this in my local Chrome testing.

Reply:
@tom-fleming Thank you for pointing it out.  I set `width: 100vw;` in /assets/scss/01-atoms/global/_elements.scss.